catalog/spatial/registry/registry-admin-modules/registry-admin-remote-ui/src/main/webapp/js/view/ModalRegistry.view.js

```diff
@@ -233,15 +234,24 @@ function (ich,Marionette,Backbone,ConfigurationEdit,wreqr,_,$,Utils,Service,moda
         },
         showError: function(msg) {
             var view = this;
-            var $group = view.$el.find('.registryName>.control-group');
-
-            $group.find('.error-text').text(msg).show();
+            var $group = view.$el.find('.registryUrl');
```

Comment:
I don't think this isn't finding the url field of a non-csw url, causing the modal to not have an error displayed, but still causing the submit button to be disabled.